Documentation/Books/Manual/Installation/Linux.md

```diff
@@ -1,101 +1,60 @@
-Linux
-=====
+Installing ArangoDB on Linux
+============================
 
 To install ArangoDB on Linux:
 
-- Visit the official [Download](https://www.arangodb.com/download) page of the
-  ArangoDB web site and download the correct package for your Linux distribution.
-  You can find binary packages for the most common distributions there.
-- Follow the instructions to use your favorite package manager for the
-  major distributions. After setting up the ArangoDB repository you can
-  easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.
-- Debian based packages will ask for a password during installation. For an
-  unattended installation for Debian, see [below](#unattended-installation).
-  Red-Hat based packages will set a random password during installation.
-  For other distributions or to change the password, run
-  `arango-secure-installation` to set a root password. 
-- Alternatively, see [Compiling](Compiling.md) if you want to build ArangoDB
-  yourself.
-  
-After installation, the ArangoDB Server can be started with a command like the
-following:
+1. Visit the official [Download](https://www.arangodb.com/download) page of the
+   ArangoDB web site and download the correct package for your Linux distribution.
+   You can find binary packages for the most common distributions there. Linux Mint: 
+   please use the corresponding Ubuntu or Debian packages. 
+1. Follow the installation instructions included in the _Download_ page to use your
+   favorite package manager for the major distributions. After setting up the ArangoDB
+   repository you can easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.
 
-    unix> /etc/init.d/arangod start
- 
-The above command will start the server, and do that as well at system boot time.
+{% hint 'info' %}
+Starting from version 3.4.0 in addition to installation packages (distribution dependent)
```

Comment:
```suggestion
In addition to installation packages (distribution dependent) a `tar.gz` archive
is available starting from version 3.4.0.
```
Does this archive actually contain a universal binary that was talked about some time ago? If so, should we mention it?

Reply:
@Simran-B I believe so. We should check with engineering and in case improve later the description we added imho